### ISSUE TITLE
[Forecasting Data] Create a service to upload the data and update the forecasting tables

### ISSUE THREAD

**Issue by luabida** [Mosqlimate-project/Data-platform#167]
Currently the CSV data is being served via [AlertaDengue/transfer-learning-forecast](https://github.com/AlertaDengue/transfer-learning-forecast/tree/main/forecast/forecast_tables) repository. The Platform should have a service to upload these files into the Mosqlimate server and a file watcher to trigger the probabilistic model when the CSVs are uploaded through this service

**Comment by fccoelho** [comment]
Just to be more specific, the upload does not need to be a part of the API, unless it is easier to do it this way. The publishing of forecasts will be an internal task, not something that users can do.

**Comment by fccoelho** [comment]
We need to advance this issue? do we already have a forecasts table template defined?

**Comment by luabida** [comment]
I've this issue almost finished, just need to fix the geochart configuration that has some issues rendering the maps due to its result weight in html

**Comment by eduardocorrearaujo** [comment]
> I've this issue almost finished, just need to fix the geochart configuration that has some issues rendering the maps due to its result weight in html

Can we close this issue?

**Comment by eduardocorrearaujo** [comment]
Acho que isso daqui era mais pensado para o forecast map, certo?

**Comment by fccoelho** [comment]
Sim, podemos fechar. pois não vamos ter uma tabela de forecasts separada, vamos usar as predictions na plataforma.
